### ISSUE TITLE
md-select doesn't read currently selected item in screen readers

### ISSUE THREAD

**Issue by calebegg** [angular/material#3891]
A native `<select>` reads aloud the currently selected item when you focus it. `<md-select>` does not. Reproduced in JAWS/Firefox and Chromevox.

This can be reproduced on the [demo page](https://material.angularjs.org/latest/#/demo/material.components.select).

**Comment by rschmukler** [comment]
@marcysutton can you help me out with this?

**Comment by marcysutton** [comment]
There are a few things that make this a challenging one to fix. First, there is no native `<input>` to put a label on....so, `aria-label` has been set to "State" on the select. But then, there is no way to announce a default value unless we append it to `aria-label`, combining the label and value into one string. Second, because the options are not present until you interact with the select and they are in a different part of the DOM, we can't reliably use `aria-activedescendant` to announce the value. 

So, I have two ideas: 

1. try jamming the default value into `aria-label` along with the actual label (updating with value changes)
2. move `md-select-menu` to a descendant of `md-select` that is always present, instead of injecting on user interaction.

The first one is a total hack, but it's a lot less risk and much more likely to get done.
